lms/djangoapps/instructor_task/tasks_helper.py

```diff
@@ -549,7 +552,53 @@ def upload_csv_to_report_store(rows, csv_name, course_id, timestamp):
     )
 
 
-def upload_grades_csv(_xmodule_instance_args, _entry_id, course_id, _task_input, action_name):
+def verification_status_for_user(user, course_id, user_enrollment_mode):
+    """
+    Returns the verification status.
+    """
+    if user_enrollment_mode not in CourseMode.VERIFIED_MODES:
+        return 'ID Verified'
+
+    user_is_verified = SoftwareSecurePhotoVerification.user_is_verified(user)
+
+    if not user_is_verified:
+        return 'Not ID Verified'
+    else:
+        user_is_re_verified = SoftwareSecurePhotoVerification.user_is_reverified_for_all(course_id, user)
+        if not user_is_re_verified:
+            return 'ID Verification Expired'
+        else:
+            return 'ID Verified'
+
+
+def certificate_info_for_user(user, course_id, whitelist_user_ids, grade):
+    """
+    Returns the certificate info for a user for a particular course.
+    """
+    user_in_embargoed_list = UserProfile.objects.filter(user=user, allow_certificate=False).exists()
+    eligible_for_certificate = not user_in_embargoed_list and (user.id in whitelist_user_ids or grade is not None)
+
+    if eligible_for_certificate:
+        is_eligible = 'Y'
+        certificate_status = certificate_status_for_student(user, course_id)
+        certificate_generated = certificate_status['status'] == CertificateStatuses.downloadable
+        is_delivered = 'Y' if not user_in_embargoed_list and (user_in_embargoed_list or certificate_generated) else 'N'
+        certificate_type = certificate_status['mode'] if certificate_generated else 'N/A'
+    else:
+        is_eligible = 'N'
+        is_delivered = 'N'
+        certificate_type = 'N/A'
+
+    # ['Certificate Eligible', 'Certificate Delivered', 'Certificate Type']
+    info = [
```

Comment:
No need for the comment and you can just return here instead of setting a variable.